IoTConnect_2.0/IoTConnect_2.0/IoTConnectManager/IoTConnectManager.swift

```diff
@@ -300,6 +319,27 @@ class IoTConnectManager {
         }
     }
     
+    //send ack command for ota,module and device command
+    func sendAckCmd(ackGuid:String,status:String, msg:String = "",childId:String = "",type:Int){
+        if dictSyncResponse.count > 0{
+            let dictToSend =  [
+                "dt":objCommon.now(),
```

Comment:
Please use cod able or use constants for keys

Reply:
@sw-ot-deveshmevada Changes are done in new push